overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/module-setup.sh

```diff
@@ -15,7 +15,10 @@ install() {
         chroot \
         sync \
         bwrap \
-        env
+        env \
+        zipl
```

Comment:
Don't we need to make this architecture conditional?  It looks like in upstream dracut there's some usage of
`case "${DRACUT_ARCH:-$(uname -m)}" in`

Reply:
Sure, will do